utils/include/eigen_utils.h

```diff
@@ -0,0 +1,41 @@
+#ifndef TORC_EIGEN_UTILS_H
+#define TORC_EIGEN_UTILS_H
+
+#include "eigen3/Eigen/Dense"
+
+namespace torc::utils {
+    /**
+     * Converts a std vector to an Eigen column vector.
```

Comment:
Can you also add `@brief` tags here too

Reply:
Of course, I also added the tags to the previously written Function classes.